opentelemetry-sdk/src/opentelemetry/sdk/metrics/export/controller.py

```diff
@@ -44,6 +44,7 @@ def run(self):
 
     def shutdown(self):
         self.finished.set()
+        self.tick()
```

Comment:
Can you add a comment explaining why we do this? (one more collection pass on exit)